code/modules/reagents/oldchem/reagents/_reagent_base.dm

```diff
@@ -21,57 +21,42 @@
 	var/process_flags = ORGANIC
 	var/admin_only = 0
 
-/datum/reagent/proc/reaction_mob(mob/M, method=TOUCH, volume) //Some reagents transfer on touch, others don't; dependent on if they penetrate the skin or not.
-	if(!istype(M, /mob/living))
```

Comment:
`reaction_mob` is only ever called from `reaction`, in blob code, and for bees---regardless of the case, they're all gated behind `isliving` checks or ONLY apply to living mobs.